### ISSUE TITLE
ONNXRuntime StandardScaler distinctions

### ISSUE THREAD

**Issue by ladodc** [microsoft/onnxruntime#4883]
Hi, 
I have saved my pipeline with scikit-learn StandardScaler into ONNX Format. After that I have inferenced this model with ONNXRuntime and got a bit different results as in scikit-learn. See my example. Why is it so?

[ONNXRuntime StandardScaler distinctions.zip](https://github.com/microsoft/onnxruntime/files/5108748/ONNXRuntime.StandardScaler.distinctions.zip)

**System information**
- OS Platform and Distribution (Win10
- ONNX Runtime version: 1.3

- Python version: 3.7

**To Reproduce**

```
import numpy
from sklearn.datasets import load_iris
from sklearn.pipeline import Pipeline
from sklearn.preprocessing import StandardScaler

data = load_iris()
X = data.data[:5, :2]
y = data.target

pipeline = Pipeline([('scaler', StandardScaler())])
pipeline.fit(X, y)
print(X)
print(pipe.transform(X)
```
[[5.1 3.5]
 [4.9 3. ]
 [4.7 3.2]
 [4.6 3.1]
 [5.  3.6]]
[[ 1.29399**328**  0.950255**27**]
 [ 0.21566555 -1.2094158 ]
 [-0.86266219 -0.34554737]
 [-1.40182605 -0.77748158]
 [ 0.75482941  1.38218948]]
```
from onnxmltools import convert_sklearn
from onnxmltools.utils import save_model
from onnxmltools.convert.common.data_types import FloatTensorType, StringTensorType

onnx_model = convert_sklearn(pipeline, 'data preprocessing',
                            [('X1', FloatTensorType([None, 1])),
                            ('X2', FloatTensorType([None, 1]))])

#Save data preprocessing model
save_model(onnx_model, 'preprocessing.onnx')

import onnxruntime as rt

sess = rt.InferenceSession("preprocessing.onnx")

label_name = sess.get_outputs()[0].name
result = sess.run([label_name], {"X1": X[:,0].reshape(5,1).astype(numpy.float32),"X2": X[:,1].reshape(5,1).astype(numpy.float32)})[0]
print(result)
```
[[ 1.29399**2**    0.950255**4** ]
 [ 0.21566534 -1.2094157 ]
 [-0.8626639  -0.34554705]
 [-1.4018273  -0.77748185]
 [ 0.7548287   1.3821892 ]]

**Comment by xadupre** [comment]
Your issue is very similar to #4130, scikit-learn is using double to compute the outputs and ONNX is using float. Even if float are used as input in scikit-learn, the coefficient of StandardScaler are stored in double. That explains the small discrepencies between the two.

I introduced an option to keep double coefficients in ONNX if you don't want any discrepency for this operator.

```
onx_div_cast = convert_sklearn(...
                       options={StandardScaler: {'div': 'div_cast'}})
```

**Comment by ladodc** [comment]
Hi Xavier, 
thanks for you suggestion. I have tried to follow the issue #4130 and the results after inferencing onnx model brings realy the same output as the output from scikit learn.
I have changed my pipeline like this:

`pipeline = Pipeline([
                     ('cast64',ColumnTransformer([
                        ('feature selection', CastTransformer(dtype=np.float64), ['X1','X2']) ])),
                     ('scaler',StandardScaler()),
                     ('cast', CastTransformer())])`

The only think I wonder is the onnx model. In all Cast operators in attribute to I see: type: Int64. This I do not undestand. I have expected float32 or float64. Isn´t a Bug?

This is my jupyter notebook: [ONNXRuntime StandardScaler distinctions.zip](https://github.com/microsoft/onnxruntime/files/5124541/ONNXRuntime.StandardScaler.distinctions.zip)
Hier ist the generated ONNX Model: [preprocessing.zip](https://github.com/microsoft/onnxruntime/files/5124516/preprocessing.zip)

![image](https://user-images.githubusercontent.com/10833738/91191589-43609300-e6f5-11ea-97d8-85d1e0d8827f.png)
